hieradata_aws/production.yaml

```diff
@@ -427,10 +429,7 @@ monitoring::checks::lb::loadbalancers: # prefer "internal"
     healthyhosts_ignore:
       - draft-fron-draft-finder-frontend # no idea what this is
   blue-email-alert-api-internal: {}
-  blue-frontend-internal:
-    healthyhosts_ignore:
-      - frontend-i-designprinciples # no idea what this is
-      - frontend-i-spotlight # no idea what this is
```

Comment:
You're right that we'll be able to delete these when these apps are no longer included in the healthchecks for the `frontend` machine. I'd maybe do this removal in a second commit, after testing that the first commit (the addition of the `ensure: 'absent'`) actually works.